modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/java/com/liferay/frontend/taglib/clay/sample/web/internal/display/context/DataSetDisplayContext.java

```diff
@@ -39,7 +39,7 @@
 	private final DataSetDisplayRequestHelper _dataSetDisplayRequestHelper;
 
 	public String getAPIURL() {
-		return "/o/object-admin/v1.0/object-definitions";
+		return "/o/c/sampleobjectdefinitions";
```

Comment:
Can we get data directly from browser? Just for testing. I tried, hoping to get sample JSON , but I got:

![screen](https://user-images.githubusercontent.com/5435511/134515320-449381ae-1d47-4b39-a214-48b40a45b81e.png)

Reply:
It's weird. I can expect an unauthorized response but not an internal server error. We might be using some specific header or parameter from the Liferay environment.

I'll research this if it is useful for you

Reply:
Don't worry about it, it's not hugely important. We can still see JSON by opening request in dev tools. This would just be a bit more convenient.